include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -424,7 +422,7 @@ class format_help : public format_help_base<format_help>
     //!\brief Needed for correct formatting while calling different print functions.
     bool prev_was_paragraph{false};
     //!\brief Stores the relevant parameters of the documentation on the screen.
-    console_layout_struct layout;
+    static inline console_layout_struct const layout{};
```

Comment:
?

Reply:
I had this super weird behaviour that when it was not const, then within print_text the values of `layout` suddenly changed :confused: Since it does not change anyway I made it const  which resolved the issue..

Reply:
> then within print_text the values of layout suddenly changed

o_O

Reply:
Did you initialise it before?

Reply:
I did, same result.